gallery/visualtests/darkModeSettings.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+const { setupBrowser } = require('./testUtils');
+
+describe('Dark Mode Settings', function() {
+  const { waitAndGetElements,
+    getPage,
+    checkFullPageScreenshot,
+    overrideDisplayTheme,
+    removeThemeEnabling
+  } = setupBrowser('#/');
+
+  const openModalBtn = '.gallery-page-header__theme-settings-button',
+      themeSettingsModal = '.gallery-theme-settings-modal';
+
+  async function openThemeSettingsModal() {
```

Comment:
Since we're no longer using the modal to drive the settings, there's really no reason to have it open for these tests

Reply:
> I went back on forth on whether including the modal was helpful or not. Makes sense to leave it out, so openThemeSettingsModal() removed here: [b01b922](https://github.com/sonatype/sonatype-react-shared-components/pull/808/commits/b01b922d9a59522d5dce573c40b883865447131d)

You'll need to update the screenshots.

Reply:
Screenshots updated here: [3d2e65f](https://github.com/sonatype/sonatype-react-shared-components/pull/808/commits/3d2e65f70b29ea49997bd5433be89411631ca285)